packages/web3-providers-http/test/fixtures/config.ts

```diff
@@ -0,0 +1,19 @@
+type TAccount = {
```

Comment:
We don't use and prefer to use any prefix for the types. 
```suggestion
type Account = {
```

Reply:
fixed